client/blocks/inline-help/index.jsx

```diff
@@ -136,6 +137,11 @@ class InlineHelp extends Component {
 		if ( ! this.props.isInlineHelpVisible ) {
 			return null;
 		}
+
+		if ( isWpMobileApp() ) {
```

Comment:
Should this be implemented in the `isInlineHelpVisible()` selector so it would be grabbed by the above selector check?

Or even better, can we avoid loading this block and all the related code by asynchronously loading it only if this isn't the mobile app?

Reply:
This should be resolved no changes are now needed to this file